### PR TITLE
Add blog post on Angular and MySQL

### PR DESCRIPTION
**Pull request by mraible**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A title that's approved by Dev Marketing
- [x] A URL that's approved by Dev Marketing
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2019-08-16-angular-mysql-express.md [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+create user 'timeline'@'localhost' identified by 'password';
+grant all on timeline.* to 'timeline'@'localhost';
+```
+
+In the first line, replace `password` with a more complex password. The rest of this tutorial assumes you used "password", so if you change it, please make sure and change it in all the code snippets below.
+
+Now you are ready to create the `events` table in the new database by specifying the data schema. 
+
+```sql
+CREATE TABLE events (
```
Comment (dogeared) [inline _source/_posts/2019-08-16-angular-mysql-express.md]: nit: you use lower case `create ...` in all the other sql commands and all caps here. not a show stopper.

File: _source/_posts/2019-08-16-angular-mysql-express.md [hunk 2]
```diff
@@ -0,0 +1,811 @@
...
+  production: false,
+  serverUrl: 'http://localhost:8080'
+};
+```
+
+The comments in `src/environments/environment.ts` provide useful hints about how to use environments in Angular.
+
+Now you are ready to implement the timeline component. Open `src/app/timeline/timeline.component.html` and edit the contents to match the code below.
+
+{% raw %}
```
Comment (dogeared) [inline _source/_posts/2019-08-16-angular-mysql-express.md]: the other `html` snippets don't have the `{% raw %}` around them. Is it necessary here?

File: _source/_posts/2019-08-16-angular-mysql-express.md (hunk 1 above)
Comment (mraible) [inline _source/_posts/2019-08-16-angular-mysql-express.md]:
```suggestion
create table events (
```

File: _source/_posts/2019-08-16-angular-mysql-express.md (hunk 2 above)
Comment (mraible) [reply _source/_posts/2019-08-16-angular-mysql-express.md]: It is because there’s double {{ }} in this block.

**Pull request merged**